card/mod/05_standard/set/all/event_viz.rb

```diff
@@ -2,10 +2,14 @@
 # EXPERIMENTAL
 # the following methods are for visualing card events
 #  not ready for prime time!
+#  not working with Rails 4
 
 def events action
   @action = action
-  root = _validate_callbacks + _save_callbacks
+
+  # don't know what this line is for but it breaks with Rails 4
+  # root = _validate_callbacks + _save_callbacks
+
```

Comment:
It's basically a developer tool.  The idea here is to be able to do mycard.events and see output of all the events that a card goes through.

I have found it occasionally useful in the past and likely will again, but it doesn't have be working for this PR.